src/matcha_ml/core/core.py

```diff
@@ -76,6 +83,41 @@ def get(
     return result
 
 
+def destroy(resources: List[Tuple[str, str]]) -> None:
+    """Destroy resources.
+
+    Args:
+        resources (List[Tuple[str,str]): the list of resources to be actioned by the verb to be provided to the user as a status message
```

Comment:
Why does the API `destroy` function **require** a parameter to construct a UI message? This should all be done in the `cli` prior to calling this function.

Reply:
Removed, thanks!